tests/integration/src/test/resources/bug-login-breaks-stream.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<mule xmlns="http://www.mulesoft.org/schema/mule/core"
```

Comment:
Ok. I tried to do a unit test for this issue. However, DefaultMuleMessage needs a MuleContext to be tested. I could reuse an existing config.